instrumentor-aop/src/main/java/com/sproutsocial/metrics/InstrumentationDetails.java

```diff
@@ -0,0 +1,60 @@
+package com.sproutsocial.metrics;
+
+import com.google.common.base.Strings;
+import org.aopalliance.intercept.MethodInvocation;
+
+import java.lang.reflect.Method;
+
+/**
+ * Created on 2/29/16.
+ */
+public interface InstrumentationDetails {
+
+    Instrumented getAnnotation(MethodInvocation methodInvocation);
+    String name(Method method, Instrumented annotation);
+
+    class ClassInstrumentation implements InstrumentationDetails {
```

Comment:
`ClassInstrumentation` prefixes `Instrumented.name()` (if present) to the `Method.getName()`, while `MethodInstrumentation` uses it on its lonesome.  Can you explain this difference?